ote/src/clj/ote/services/admin.clj

```diff
@@ -42,23 +42,35 @@
 
 (defn- list-users [db user query]
   (nap-users/list-users db {:email (str "%" query "%")
-                            :name (str "%" query "%")}))
+                            :name  (str "%" query "%")}))
+(defn- published-search-param [query]
+  (cond
+    (or (nil? (:published-type query)) (= :ALL (:published-type query))) nil
```

Comment:
wouldn't case be simpler?
```
(case (:published-type query)
  :YES {::t-service/published? true}
  :NO {::t-service/published? false}
  nil)
```

Even simpler if we just used `true`/`false` instead of `:YES`/`:NO` keywords

Reply:
Even simpler: 
```
(when-let [pub (:published-type query)]
  {::t-service/published? 
   (case pub
    :YES true
    :NO false)})
```

Reply:
:published-type can also be nil or :ALL. So there is 4 possibilities.

Reply:
nil is possible, because material ui can't set default values for dropdownlist by default. It can be done by some sorcery, but I didn't want use anymore time to these admin view.